docs/source/api/v1/divisions_id.rst

```diff
@@ -88,13 +88,16 @@ Response Structure
 	Date: Thu, 29 Nov 2018 19:59:57 GMT
 	Content-Length: 78
 
-	{ "response": [
-		{
+	{ "alerts": [{
+			"text": "This endpoint is deprecated, please use GET /divisions with the 'id' parameter instead",
+			"level": "warning
```

Comment:
@shamrickus: Getting a warning building the docs because of no closing quote.

```
~/go/src/github.com/apache/trafficcontrol/docs/source/api/v1/divisions_id.rst:77: WARNING: Could not lex literal_block as "http". Highlighting skipped.
```

Reply:
Fixed in #4476